From 0509446aba99622dbef8ad70c6e69209ce67fe64 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 14 Jun 2021 08:54:15 +0000
Subject: [PATCH] libstdc++: fix ostream operator<< usage in
 JsepTransportCollection

it.second in JsepTransportCollection::IsConsistent() is of type
std::unique_ptr, which has no operator<< in libstdc++. Even if it
would exist, it would just return the pointer value and not the
content.
---
 third_party/webrtc/pc/jsep_transport_collection.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/webrtc/pc/jsep_transport_collection.cc b/third_party/webrtc/pc/jsep_transport_collection.cc
index 96cbd92..03b85a4 100644
--- a/third_party/webrtc/pc/jsep_transport_collection.cc
+++ b/third_party/webrtc/pc/jsep_transport_collection.cc
@@ -197,7 +197,7 @@ bool JsepTransportCollection::IsConsistent() {
   for (const auto& it : jsep_transports_by_name_) {
     if (!TransportInUse(it.second.get())) {
       RTC_LOG(LS_ERROR) << "Transport registered with mid " << it.first
-                        << " is not in use, transport " << it.second;
+                        << " is not in use, transport " << it.second.get();
       return false;
     }
     const auto& lookup = mid_to_transport_.find(it.first);
-- 
2.31.1

